scripts/zones/AlTaieu/DefaultActions.lua

```diff
@@ -1,5 +0,0 @@
-local ID = require("scripts/zones/AlTaieu/IDs")
```

Comment:
Just leaving this here before I have time to review.  This file should _not_ be removed, and that is indeed the default action for _0x0 prior to having access.

Reply:
I understand this, but I have moved the default action into the actual mission script based on when the player gains access to Al'Taieu. Is there a specific reason we need this in a separate lua?